docs/securing_fledge.rst

```diff
@@ -142,11 +148,17 @@ The user management pages allows
   - Deleting users.
   - Resetting user passwords.
   - Changing the role of a user.
+  - Changing the details of a user
+
+Fledge currently supports four roles for users:
+
+  - **Administrator**: a user with admin role is able to fully configure Fledge, view the data read by the Fledge instance  and also manage Fledge users
 
-Fledge currently supports two roles for users,
+  - **Editor**: a user with this role is able to configure Fledge and view the data read by Fledge. The user can not manage other users or add new users.
```

Comment:
`A user` - caps (A)

Reply:
We don't do this anywhere else when using hanging ident tags like this.